packages/story-editor/src/components/canvas/editLayerFocusManager/constants.js

```diff
@@ -15,6 +15,7 @@
  */
 const FOCUS_GROUPS = {
   EDIT_ELEMENT: 'EDIT_ELEMENT',
+  EDIT_CANVAS: 'EDIT_CANVAS',
```

Comment:
nit: just `CANVAS`? I think `EDIT_ELEMENT` was because it's the actively being edited element.

Reply:
That or `ELEMENT_SELECTION` because eventually we'll have a `PAGE_SELECTION` group I think?